domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/ReviewSchedule.kt

```diff
@@ -51,6 +51,7 @@ enum class ReviewScheduleStatus(val isExclusion: Boolean = false, val isExemptio
   EXEMPT_PRISONER_TRANSFER(isExemption = true),
   EXEMPT_PRISONER_RELEASE(isExemption = true),
   EXEMPT_PRISONER_DEATH(isExemption = true),
+  EXEMPT_UNKNOWN,
```

Comment:
New exemption status - couldnt think of a better name 🤣

Reply:
do Meganexus need to know this status? If they do then we'll need to update the integration api (docs only I think) to tell them that it is a possibility.